lego/apps/articles/fixtures/development_articles.yaml

```diff
@@ -56,3 +53,4 @@
     tags:
       - lorem
       - ipsum
+    youtube_url: https://www.youtube.com/watch?v=bLHL75H_VEM
```

Comment:
Do we need both articles in dev fixtures to have youtube headers? Could be annoying when developing other things. Maybe add a third article for youtube dev work, instead of changing the existing ones?

Reply:
For sure!

Reply:
Good idea! 👍